js.go

```diff
@@ -2813,15 +2831,36 @@ func (sub *Subscription) Fetch(batch int, opts ...PullOpt) ([]*Msg, error) {
 			nr.Expires = expires
 			nr.NoWait = noWait
 			nr.MaxBytes = o.maxBytes
+			if 2*o.hb < expires {
```

Comment:
Unfortunately this is required because Fetch can send more than 1 pull request, and even though for the first request the value may be valid, when the second one is sent we don't really know up front what the expiry will be - so if we depend on the server we will return an error when we shouldn't.

Reply:
Hm. Maybe return error when users tries to send fetch with bad hb?

Reply:
👍